### PR TITLE
Adding labs and pics done by Nigel

### PR DESCRIPTION
**Pull request by nigelpoulton**
@ManoMarks Here are the lab files. 
- No MACVLAN lab as this can't be done in AWS/Azure etc...
- I've included them as-is, and added the image files to the `img` directory.
- I've not changed the lab's landing page as I'm not sure how you want to link to these labs.
Signed-off-by: Nigel <nigelpoulton@hotmail.com>

### CODE REVIEW

File: networking/concepts/A4-HTTP Routing Mesh.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# HTTP Routing Mesh (HRM)
+
+> **NOTE TO MIKE/BETTY:** This lab assumes two things.
```
Comment (ManoMarks) [inline networking/concepts/A4-HTTP Routing Mesh.md]: Let's turn this into a generic instruction. Leave it up to them but suggest the local host file if doing it locally. Also remove "Mike/Betty" :-)

File: networking/concepts/A4-HTTP Routing Mesh.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+You will need all of the following to complete this lab:
+
+- A UCP Cluster running **Docker 1.12** or higher
+- Credentials to log in to UCP, create services, and enable the HRM
+- Name resolution configured for two DNS names (your lab instructor will give you these)
+
+Your instructor will provide you with the details you require.
+
+> **MIKE** This next note can be removed if you go with local name resolution for red.example.com and white.example.com.
```
Comment (ManoMarks) [inline networking/concepts/A4-HTTP Routing Mesh.md]: Remove this line

File: networking/concepts/A4-HTTP Routing Mesh.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+> **NOTE:** Throughout this guide we will use *red.example.com* and *white.example.com*. The DNS names in your lab will be different and your instructor will provide you details. This means that any time you see *red.example.com* and *white.example.com* you will need to substitute these for *red.<your-domain-goes-here>* and *white.your-domain-goes-here>*.
```
Comment (ManoMarks) [inline networking/concepts/A4-HTTP Routing Mesh.md]: Change this to:
+> **NOTE:** Throughout this guide we will use *red.example.com* and *white.example.com*. As per the note above, you will need to configure this yourself. You can also substitute other domain names if you like.  That would mean that any time you see *red.example.com* and *white.example.com* you will need to substitute these for *red.<your-domain-goes-here>* and *white.your-domain-goes-here>*.

File: networking/concepts/A4-HTTP Routing Mesh.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+3. Verify that both services are up and running by clicking `Resources` > `Services` and checking that both services are running as shown below.
+
+  ![](img/check-svc.png)
+
+You now have two services running. Both are connected to the `ucp-hrm` network and both have the `com.docker.ucp.mesh.http` label. The **RED** service is associated with HTTP requests for `red.example.com` and the **WHITE** service is associated with HTTP requests for `white.example.com`. This mapping of labels to URLs is leveraged by the `ucp-hrm` service which is published on port 80.
+
+
+# <a name="test"></a>Step 5: Test the configuration
+
+> **NOTE TO MIKE: DNS name resolution is required for this step. This can obviously be via the local hosts file, but this step will not work unless the URLs specified in the `com.docker.ucp.mesh.http` labels resolve to the UCP cluster nodes (probably via a load balancer).**
```
Comment (ManoMarks) [inline networking/concepts/A4-HTTP Routing Mesh.md]: Remove "TO MIKE"

**Comment by nigelpoulton** [comment]
Made changes and pushed. LMK if you need anything else @ManoMarks

**Pull request merged**